searx/engines/ebay.py

```diff
@@ -20,8 +20,10 @@
 categories = ['shopping']
 paging = True
 
-url = 'https://www.ebay.com'
-search_url = url + '/sch/i.html?_nkw={query}&_sacat={pageno}'
+# Set base_url in settings.yml in order to
+# have the desired local TLD.
+base_url = None
```

Comment:
Can you either set a default URL here, or set the base_url in settings.yml ?

So the engine works with the default configuration.